### PR TITLE
fix: declaration compatibility of ParentRuntime::createProcess #43

### PR DESCRIPTION
**Pull request by peteradeojo**
Fixed the issue in #43 .

### CODE REVIEW

**Comment by vuongxuongminh** [comment]
> Fixed the issue in #43 .

This changed incompatible with old versions of async, you should bump spatie async version to latest version.

**Comment by peteradeojo** [comment]
spatie/async is already at the latest. 1.6.0

**Comment by vuongxuongminh** [comment]
I mean you should update the version of Spatie async in `composer.json` to `^1.6`, currently this package requires `^1.5` so if our users using version 1.5, this change will cause an error.

**Comment by peteradeojo** [comment]
> Fixed the issue in #43 .

**Comment by peteradeojo** [comment]
Version updated to 1.6

Sorry for the closing and reopening. I was trying to add a comment. Not really familiar with this open source of a thing

File: src/Runtime/ParentRuntime.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
  */
 class ParentRuntime extends BaseParentRuntime
 {
-    public static function createProcess($task, ?int $outputLength = null, ?string $binary = 'php'): Runnable
+    public static function createProcess($task, ?int $outputLength = null, ?string $binary = 'php', ?int $max_input_size = 100000): Runnable
     {
         $runnable = parent::createProcess($task, $outputLength, $binary);
```
Comment (vuongxuongminh) [inline src/Runtime/ParentRuntime.php:26]: Please forward max imput arg to parent method.

**Review by vuongxuongminh — CHANGES_REQUESTED** [review]

File: src/Runtime/ParentRuntime.php (hunk 1 above)
Comment (peteradeojo) [reply src/Runtime/ParentRuntime.php]: Done

**Review by vuongxuongminh — APPROVED** [review]

**Comment by vuongxuongminh** [comment]
Thank you @peteradeojo

**Comment by peteradeojo** [comment]
> Thank you @peteradeojo 

Thank you too for this package. I use it at work so it's important to me lol

**Pull request merged**